deploy/infra/master/replicaset/deployment.yaml

```diff
@@ -116,6 +128,18 @@ spec:
           - name: k8s-azure-file
             mountPath: /etc/kubernetes/azure.json
             readOnly: true
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 10
+          periodSeconds: 5
+        readinessProbe:
```

Comment:
`ReadinessProbe` is typically used to see if an endpoint is ready to receive traffic. Only when the readinessprobe passes the pod ip is added to list of endpoints in the service that is exposing the pod. We don't have a service with mic or nmi. Is there a reason we are doing readiness probe too?